### PR TITLE
Added stylesheets for ant-design package and added docs in readme

### PR DESCRIPTION
**Pull request by kevinrodriguez-io**
- This commit moves some styles into their own stlyesheet and also includes classNames where appropiate.
- Also, added some documentation on the readme file on how to install and where to check usage.

### CODE REVIEW

File: packages/ant-design/README.md [hunk 1]
```diff
@@ -1,5 +1,65 @@
...
+.ant-design-wallet-icon
+.ant-design-wallet-modal-menu
+.ant-design-wallet-modal-menu-item
+.ant-design-wallet-modal-menu-button
+.ant-design-wallet-modal-menu-button-icon
+.ant-design-wallet-multi-button-menu
+.ant-design-wallet-multi-button-menu-wallet-name-item
+.ant-design-wallet-multi-button-menu-wallet-name-button
+.ant-design-wallet-multi-button-icon
+.ant-design-wallet-multi-button-menu-item
```
Comment (jordansexton) [inline packages/ant-design/README.md:64]: Can we change `ant-design-wallet-` to `wallet-adapter-` as a prefix?

File: packages/ant-design/src/styles.less [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+.ant-design-wallet-multi-button-menu {
+    padding   : 0px;
+    margin-top: -44px;
+}
+
+.ant-design-wallet-multi-button-menu-wallet-name-item {
+    padding: 0px;
+}
+
+.ant-design-wallet-multi-button-menu-wallet-name-button {
```
Comment (jordansexton) [inline packages/ant-design/src/styles.less:46]: Would be nice to shorten some of these names --

`menu-wallet-name-item` -> `menu-item`
`menu-wallet-name-button` -> `menu-button`

**Review by jordansexton — COMMENTED** [review]
Looks good overall. The long CSS classes are a bit unwieldy, perhaps we can shorten them a bit? I think if there's a `wallet-adapter` prefix on them, we can eliminate a number of the other qualifiers.

**Comment by kevinrodriguez-io** [comment]
Good notes on the naming! Alright, refactor done!

**Comment by jordansexton** [comment]
LGTM 🚀

**Pull request merged**